app/presenters/curation_concerns/inspect_work_presenter.rb

```diff
@@ -0,0 +1,66 @@
+module CurationConcerns
+  class InspectWorkPresenter
+    def initialize(solr_document, current_ability)
+      @solr_document = solr_document
+      @current_ability = current_ability
+    end
+    attr_reader :solr_document, :current_ability
+
+    def workflow
+      workflow_resp = { comments: [], roles: [] }
+      if sipity_entity
+        workflow_resp =
+          {
+            entity_id: sipity_entity.id,
+            proxy_for: sipity_entity.proxy_for_global_id,
+            workflow_id: sipity_entity.workflow_id,
+            workflow_name: sipity_entity.workflow_name,
+            state_id: sipity_entity.workflow_state_id,
+            state_name: sipity_entity.workflow_state_name,
+            comments: workflow_comments,
+            roles: sipity_entity_roles
+          }
+      end
+      workflow_resp
+    end
+
+    def solr
+      @solr_document.inspect
+    end
+
+    private
+
+      def sipity_entity
+        @sipity_entity ||= PowerConverter.convert(solr_document, to: :sipity_entity)
+      rescue PowerConverter::ConversionError
```

Comment:
You should not catch this error.  If you do, you'll end up with null pointer errors, which are much harder to diagnose than the underlying `ConversionError`